Generator.Equals/EqualsGenerator.cs

```diff
@@ -33,6 +33,8 @@ public void Execute(GeneratorExecutionContext context)
                 context.Compilation.GetTypeByMetadataName("Generator.Equals.CustomEqualityAttribute")!
             );
 
+            var handledSymbols = new List<string>();
```

Comment:
Swapping List<string> with HashSet<string> is an easy win here -- would be noticeable if someone is dealing with thousands of [Equatable] types.

Reply:
changed List to HashSet